clusterloader2/pkg/measurement/common/slos/prometheus_measurement.go

```diff
@@ -60,6 +61,10 @@ func (m *prometheusMeasurement) Execute(config *measurement.MeasurementConfig) (
 		return nil, nil
 	}
 
+	if !m.gatherer.IsEnabled(config) {
+		klog.Warningf("%s is disabled, skipping the measuerment", m)
```

Comment:
should be some return here :)

Reply:
Also, for consistency maybe have the same log as here:
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/pkg/measurement/common/probes/probes.go#L85

Reply:
Added. 

Left the message as it was - this is general piece of code, not tied a specific measurement. I'd rather not add Kubemark specific message in here.